ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -1120,4 +1120,55 @@ ActorScope,string,Optional,ProcessEvent,,,
+EventResult,string,Mandatory,UserManagement,Enumerated,Success|Failure,
+EventSeverity,string,Mandatory,UserManagement,Enumerated,Informational|Low|Medium|High,
+EventSchema,string,Mandatory,UserManagement,,UserManagement,
+EventSchemaVersion,string,Mandatory,UserManagement,SchemaVersion,,
+ActingAppId,string,Optional,UserManagement,,,
+ActiveAppName,string,Optional,UserManagement,,,
+ActingAppType,string,Optional,UserManagement,Enumerated,Process|Browser|Resource|Other,
+HttpUserAgent,string,Optional,UserManagement,Useragent,,
+ActorUsername,string,Mandatory,UserManagement,Username,,
+ActorUserIdType,string,Conditional,UserManagement,Enumerated,SID|UID|AADID|OktaId|AWSId,ActorUserId
+ActorUserId,string,Optional,UserManagement,,,
+ActorSessionId,string,Optional,UserManagement,,,
+EventSubType,string,Optional,UserManagement,Enumerated,UserRead|UserCreated|GroupCreated|UserModified|GroupModified,
+GroupId,string,Optional,UserManagement,,,
+GroupIdType,string,Conditional,UserManagement,Enumerated,SID|UID,GroupId
+GroupName,string,Optional,UserManagement,,,
+GroupNameType,string,Conditional,UserManagement,Enumerated,UPN|Windows|DN|Simple,GroupName
+GroupType,string,Optional,UserManagement,Enumerated,Local Distribution|Local Security Enabled|Global Distribution|Global Security Enabled|Universal Distribution|Universal Security Enabled|Other,
+GroupOriginalType,string,Optional,UserManagement,,,
+SrcFQDN,string,Optional,UserManagement,FQDN,,
+SrcDvcId,string,Optional,UserManagement,,,
+SrcDvcScopeId,string,Optional,UserManagement,,,
+SrcDvcScope,string,Optional,UserManagement,,,
+SrcDvcIdType,string,Optional,UserManagement,Enumerated,AzureResourceId|MDEid,
+SrcDeviceType,string,Optional,UserManagement,Enumerated,Computer|Mobile Device|IOT Device|Other,
+SrcGeostring,string,Optional,UserManagement,,,
+SrcGeostring,string,Optional,UserManagement,,,
+SrcGeoCity,City,Optional,UserManagement,City,,
+SrcGeoLatitude,Latitude,Optional,UserManagement,,,
+SrcGeoLongitude,Longitude,Optional,UserManagement,,,
+TargetUserId,string,Optional,UserManagement,,,
+TargetUserIdType,string,Conditional,UserManagement,Enumerated,SID|UID|AADID|OktaId|AWSId,TargetUserId
+TargetUsername,string,Optional,UserManagement,Username,,
+TargetUsernameType,string,Conditional,UserManagement,Enumerated,UPN|Windows|DN|Simple,TargetUsername
+TargetUserType,string,Optional,UserManagement,Enumerated,Regular|Machine|Admin|System|Application|Service Principal|Other,
+TargetOriginalUserType,string,Optional,UserManagement,,,
+PreviousPropertyValue,string,Optional,UserManagement,,,
+NewPropertyValue,string,Optional,UserManagement,,,
+EventResultDetails,string,Recommended,UserManagement,Enumerated,NotAuthorized|Other,
+Src,string,Recommended,UserManagement,,,
```

Comment:
Src is an Alias

Reply:
Based on the documentation this field is not an Alias, but **might** Alias other fields:
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-user-management#eventsubtype:~:text=Description-,Src,alias%20the%20SrcDvcId%2C%20SrcHostname%2C%20or%20SrcIpAddr%20fields.%0A%0AExample%3A%20192.168.12.1,-SrcIpAddr

Updated per request, bu not sure if I need to follow the documentation or the not.
Please advise @devikamehra

Reply:
We need to fix the documentation in this case. Src is an alias.